release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -51,6 +51,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-9-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-9-install-ign-removed"]
+==== Installation Ignition config is removed upon boot
+
+Nodes installed with the `coreos-installer` command previously retained the installation Ignition config in the `/boot/ignition/config.ign` file. Starting with the {product-title} 4.9 installation image, that file is removed when the node is provisioned. This change does not affect clusters that were installed on previous {product-title} versions, which were thus using an older bootimage.
```

Comment:
`the coreos-installer command` might be a bit misleading, since the `coreos.inst` kernel arguments are also affected.

`which were thus` - they still are using an older bootimage, including for newly-installed nodes.

Reply:
Thanks for clarifying, @bgilbert. PTAL where I've updated the following:

- Changed `command` to `program` in the first sentence.

- In the last sentence, reworded to what I now hope is an accurate reflection of the workflow. Let me know if not.